pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -76,16 +78,23 @@ const (
 )
 
 type CheckWrapper struct {
-	ID          string                       `yaml:"id" json:"id"`
-	Text        string                       `json:"d"`
-	Type        string                       `json:"-"`
-	Remediation string                       `json:"r"`
-	State       State                        `json:"s"`
-	Scored      bool                         `json:"-"`
-	Result      map[kb.State]map[string]bool `json:"-"`
-	NodeType    []NodeType                   `json:"t"`
-	NodesMap    map[string]bool              `json:"-"`
-	Nodes       []string                     `json:"n,omitempty"`
+	ID             string                       `yaml:"id" json:"id"`
+	Text           string                       `json:"d"`
+	Type           string                       `json:"-"`
+	Remediation    string                       `json:"r"`
+	State          State                        `json:"s"`
+	Scored         bool                         `json:"-"`
+	Result         map[kb.State]map[string]bool `json:"-"`
+	NodeType       []NodeType                   `json:"t"`
+	NodesMap       map[string]bool              `json:"-"`
+	Nodes          []string                     `json:"n,omitempty"`
+	Audit          string                       `json:"a"`
+	AuditConfig    string                       `yaml:"ac"`
+	TestInfo       []string                     `json:"ti"`
+	Commands       []*exec.Cmd                  `json:"c"`
+	ConfigCommands []*exec.Cmd                  `json:"cc"`
+	ActualValue    string                       `json:"av"`
+	ExpectedResult string                       `json:"er"`
```

Comment:
I'm a little confused about the Go tags here. Is this supposed to be a short form for the tags listed above?

Reply:
Yes I think this is intended to be short-form so that the json file generated by the summarizer is not too huge, this file is anyway intermittent data to be converted by report.go in the rancher report format.
But I will just change them to all json